### ISSUE TITLE
install node fails because of ncurses missing lib

### ISSUE THREAD

**Issue by coderofsalvation** [skycocker/chromebrew#565]
On my acer chromebook 14 (chromeos 59.0.3054.0 (Official Build) dev (64-bit)) the ncurses build cannot find /usr/lib64/libmvec_nonshared.a:

```
$ uname -a
Linux localhost 3.18.0-14511-g55d103f4e8cd #1 SMP PREEMPT Wed Mar 29 19:08:08 PDT 2017 x86_64 Intel(R) Celeron(R) CPU N3160 @ 1.60GHz GenuineIntel GNU/Linux
$ crew install node
......
gcc -I. -I. -I../test -DHAVE_CONFIG_H -I. -I../include  -D_GNU_SOURCE -DNDEBUG -fPIC  --param max-inline-insns-single=1200  -fPIC -c ../test/view.c -o ../obj_s/view.o
gcc -I. -I. -I../test -DHAVE_CONFIG_H -I. -I../include  -D_GNU_SOURCE -DNDEBUG -fPIC  --param max-inline-insns-single=1200  -fPIC -c ../test/worm.c -o ../obj_s/worm.o
gcc -I. -I. -I../test -DHAVE_CONFIG_H -I. -I../include  -D_GNU_SOURCE -DNDEBUG -fPIC  --param max-inline-insns-single=1200  -fPIC -c ../test/xmas.c -o ../obj_s/xmas.o
gcc -fPIC  --param max-inline-insns-single=1200  -o background ../obj_s/background.o -Wl,-rpath,/usr/local/tmp/crew/ncurses-6.0.tar.gz.dir/ncurses-6.0/lib  -I. -I. -I../test -DHAVE_CONFIG_H -I. -I../include  -D_GNU_SOURCE -DNDEBUG -fPIC  --param max-inline-insns-single=1200  -fPIC  `echo "-L../lib -lformw -lmenuw -lpanelw -lncursesw   " | sed -e 's/-lform.*-lpanel[^ ]*//'` -lutil -lm
gcc -fPIC  --param max-inline-insns-single=1200  -o blue ../obj_s/blue.o -Wl,-rpath,/usr/local/tmp/crew/ncurses-6.0.tar.gz.dir/ncurses-6.0/lib  -I. -I. -I../test -DHAVE_CONFIG_H -I. -I../include  -D_GNU_SOURCE -DNDEBUG -fPIC  --param max-inline-insns-single=1200  -fPIC  `echo "-L../lib -lformw -lmenuw -lpanelw -lncursesw   " | sed -e 's/-lform.*-lpanel[^ ]*//'` -lutil -lm
/usr/local/lib/gcc/x86_64-cros-linux-gnu/4.9.x/../../../../x86_64-cros-linux-gnu/bin/ld: cannot find /usr/lib64/libmvec_nonshared.a
collect2: error: ld returned 1 exit status
make[1]: *** [background] Error 1
make[1]: *** Waiting for unfinished jobs....
/usr/local/lib/gcc/x86_64-cros-linux-gnu/4.9.x/../../../../x86_64-cros-linux-gnu/bin/ld: cannot find /usr/lib64/libmvec_nonshared.a
collect2: error: ld returned 1 exit status
make[1]: *** [blue] Error 1
make[1]: Leaving directory `/usr/local/tmp/crew/ncurses-6.0.tar.gz.dir/ncurses-6.0/test'
make: *** [all] Error 2
ncursesw failed to install: `make -j4` exited with 2
```

any ideaas?

**Comment by finwo** [comment]
The same thing happened on my Acer ChromeBook 14. The proposed fix in #562 should work for you.

**Comment by lyxell** [comment]
Closing as resolved, please reopen if the issues persist.
